src/raps/actions/depositCompound.ts

```diff
@@ -47,19 +47,34 @@ const depositCompound = async (
   const tokenToDeposit = requiresSwap ? outputCurrency : inputCurrency;
 
   const { accountAddress, network } = store.getState().settings;
-  const { gasPrices, selectedGasPrice } = store.getState().gas;
+  const { gasFeesBySpeed, selectedGasFee } = store.getState().gas;
   logger.log(`[${actionName}] amount`, amountToDeposit);
   const rawInputAmount = convertAmountToRawAmount(
     amountToDeposit,
     tokenToDeposit.decimals
   );
   logger.log(`[${actionName}] raw input amount`, rawInputAmount);
 
-  let gasPrice = selectedGasPrice?.value?.amount;
-  if (!gasPrice) {
-    gasPrice = get(gasPrices, `[${gasUtils.FAST}].value.amount`);
+  let maxFeePerGas = get(selectedGasFee, `gasFeeParams.maxFeePerGas.amount`);
+  let maxPriorityFeePerGas = get(
+    selectedGasFee,
+    `gasFeeParams.maxPriorityFeePerGas.amount`
+  );
+  if (!maxFeePerGas) {
+    maxFeePerGas = get(
+      gasFeesBySpeed,
```

Comment:
According to entities, `gasFeesBySpeed` does not have `gasFeeParams` (only `selectedGasFee` has this)